lib/rules/value-list-comma-newline-after/index.js

```diff
@@ -26,12 +27,49 @@ const rule = function(expectation) {
       return;
     }
 
+    let fixData;
     valueListCommaWhitespaceChecker({
       root,
       result,
       locationChecker: checker.afterOneOnly,
-      checkedRuleName: ruleName
+      checkedRuleName: ruleName,
+      fix: context.fix
+        ? (declNode, index) => {
+            const valueIndex = declarationValueIndex(declNode);
+            if (index <= valueIndex) {
+              return false;
+            }
+            fixData = fixData || new Map();
+            const commaIndices = fixData.get(declNode) || [];
+            commaIndices.push(index);
+            fixData.set(declNode, commaIndices);
+            return true;
+          }
+        : null
     });
+    if (fixData) {
+      fixData.forEach((commaIndices, decl) => {
+        commaIndices
+          .sort()
```

Comment:
There was a bug in `value-list-comma-space-after`, can you make the same change?
https://github.com/stylelint/stylelint/pull/3621/files#diff-812c870b12495af18b84bb163acd432bL54